### ISSUE TITLE
Explain this line

### ISSUE THREAD

**Issue by CarlosMenacho** [GitUser2004/second_test_robotic#2]
https://github.com/GitUser2004/second_test_robotic/blob/e111975942f300259fa7d66ee9293a837b8c702b/p1/src/dqn_robot_nav/dqn_robot_nav/dqn_agent.py#L105

**Comment by GitUser2004** [comment]
this is the return of the function replay, this function that train the neural network, and returns the loss function so this is defined by the value of the action **q_current** and the cuadratic predicted **q_network.predict(states))**2)**, so this send the error of the predictions versus the reality that was obtained
